app/views/smart_proxies/index.html.erb

```diff
@@ -8,7 +8,8 @@
     <tr>
       <th class="col-md-2"><%= s_("SmartProxy|Name") %></th>
       <th class="col-md-4"><%= s_("SmartProxy|Url") %></th>
-      <th class="col-md-4"><%= _("Features") %></th>
+      <th class="col-md-4"><%= _('Locations') %></th>
```

Comment:
That header & column should be hidden when Locations is disabled, otherwise it just shows 'None found'